web/src/left_sidebar_navigation_area.ts

```diff
@@ -11,18 +14,30 @@ import * as people from "./people.ts";
 import * as resize from "./resize.ts";
 import * as scheduled_messages from "./scheduled_messages.ts";
 import * as settings_config from "./settings_config.ts";
+import * as starred_messages from "./starred_messages.ts";
 import * as ui_util from "./ui_util.ts";
 import * as unread from "./unread.ts";
+import {user_settings} from "./user_settings.ts";
 
 let last_mention_count = 0;
 const ls_key = "left_sidebar_views_state";
 const ls = localstorage();
+export let current_active_fragment: string | undefined;
+
+const CONDENSED_PRIORITY_SCORE = 2;
+const PINNED_VIEW_SCORE = 1;
+export const UNPINNED_VIEW_SCORE = 0;
+const HIDDEN_VIEW_SCORE = -1;
```

Comment:
Let's group them together inside an object, makes it easier to expand things down the line.


```suggestion
const VIEW_PRIORITY = {
    CONDENSED: 2,
    PINNED: 1,
    UNPINNED: 0,
    HIDDEN: -1,
} as const;
```